main.go

```diff
@@ -3,16 +3,15 @@ package main
 import (
 	"github.com/alechewitt/code-wall/api"
 	"github.com/gorilla/mux"
-	"mime"
 	"net/http"
 	"os"
-	"path/filepath"
 )
 
 func main() {
 	r := mux.NewRouter()
 	r.HandleFunc("/", indexHandler)
-	r.HandleFunc("/dist/{folder}/{filename}", fileServer)
+	r.PathPrefix("/dist/").Handler(staticHandler("/dist/", "./dist/"))
+	r.PathPrefix("/node_modules/").Handler(staticHandler("/node_modules/", "./node_modules/"))
```

Comment:
Yes your right. I'll remove the node_modules. I forgot we were going to serve the code mirror stuff from a CDN. I will look at doing that on the front end branch.